dev-python/pyqt-distutils/pyqt-distutils-0.7.3.ebuild

```diff
@@ -3,7 +3,7 @@
 
 EAPI=6
 
-PYTHON_COMPAT=( python3_6 )
+PYTHON_COMPAT=( python3_{6..9} )
 inherit distutils-r1
 
 DESCRIPTION="distutils extension to work with PyQt applications and UI files"
```

Comment:
Can't get github to get right line, but the explicit setuptools dep shouldn't be needed anymore.

Reply:
Seems to be done, thank you for pointing it